src/main/java/com/code_intelligence/jazzer/junit/BUILD.bazel

```diff
@@ -40,6 +40,7 @@ java_library(
         "//src/main/java/com/code_intelligence/jazzer/driver:opt",
         "@maven//:org_junit_jupiter_junit_jupiter_api",
         "@maven//:org_junit_jupiter_junit_jupiter_params",
+        "@maven//:org_junit_platform_junit_platform_launcher",
```

Comment:
We should move this to `runtime_deps` as its not used during compilation and add a comment along the lines of "The JUnit launcher that is part of the Jazzer driver needs this on the classpath to run an @FuzzTest with JUnit".